content/blog/jumpy-0.7-current-status.md

```diff
@@ -0,0 +1,99 @@
+
+### De-Syncs
+
+There are still occasional de-syncs between network players. This means that one player is playing the game and seeing one thing, while another player is playing and seeing something different. This is caused by subtle non-determinism in the gameplay code.
+
+If you've been following Jumpy development, you might know that we specifically designed the [Bones ECS][becs] used in Jumpy to prevent non-determinism issues, so where is the non-determinism coming from!
+
+Even though the Bones ECS is deterministic, not everything in Rust is. For instance, simply storing items in a HashMap and iterating over them is non-deterministic. So it's very easy, if you're not careful, to accidentally make the game non-deterministic, just by using a HashMap! We've already fixed a couple issues like this one, and it just remains to track down what code is causing the remaining de-syncs.
+
+This is an area where we might implement new features to help the debugging process. Possibilities include automatically detecting network de-syncs by hashing the game state, and network session record/playback tools to help isolate the causes of network de-syncs.
+
+## Modding
+
+Modding is important to us. By being Open Source, anybody can change Jumpy as much as they could ever want, but we want to go further, hoping for a powerful modding ecosystem in the future.
+
+### Map Editing
+
+The easiest thing to mod in Jumpy right now is maps! Jumpy has a built-in map editor, with live editing capability, meaning you can play the game _while_ you edit the maps. All of the maps in Jumpy were made using the editor, and you can edit and save your own maps without leaving the game.
+
+We've also just completed a new map randomization feature, included in the latest nightly release, thanks to contributions from [@AustinHellerRepo]. This takes an existing map, splits it automatically into chunks, and shifts the chunks around randomly to create a new map.
+
+This is the first foray into all kinds of different possibilities for map generation and editing tools, and it laid the groundwork for implementing other map randomizer and generation algorithms. For instance, we could attempt to integrate AI map generation from text prompts using something similar to [mario-gpt](https://github.com/shyamsn97/mario-gpt).
+
+### Asset Editing
+
+In addition to maps, assets in Jumpy can be edited by modifying simple image and YAMl files. You can create characters skins, map tilesets, modify weapon sprites and create your own modifications. This works works by modifying parameters such as fuse time, explosion radius, etc.
+
+We haven't implemented mod packages yet, so there's no way to install or enable specific mods. The only way to mod the game is to modify the files in the built-in asset directory. This is absolutely something that we will improve later, with the intent to have an online mod store.
```

Comment:
> online mod store.

Call it online mod *directory*. We don’t yet know what shape our mod marketplace will take, so calling it a store is a bit too specific.